k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -297,23 +296,13 @@ function deleteOpenLdap()
     ## This function is useful for deployments prior to v0.2.0
     ## that need this cleanup method.
 
-    openldap_array=[]
-    count=0
-    while IFS= read -r line; do
-        IFS=" " read -r -a my_array <<< "$line"
-        openldap_array[count]="${my_array[1]}"
-        count=$((count+1))
-    done <<< "$(kubectl get pods -A | grep 'openldap-')"
-
-    if [ ${#openldap_array[@]} -ne 0 ]; then
-        for openldap_pod_name in "${openldap_array[@]}"
-        do
-            kubectl exec -ti "${openldap_pod_name}" --namespace="default" -- bash -c \
-                'rm -rf /etc/3rd-party/* /var/data/3rd-party/* /var/log/3rd-party/*'
-        done
+    kubectl get pods --namespace=default --output=name | grep '^pod/openldap-' |
+    while IFS= read -r pod; do
+        kubectl exec "${pod}" --namespace=default -- \
+            bash -c 'rm -rf /etc/3rd-party/* /var/data/3rd-party/* /var/log/3rd-party/*'
+    done
 
-        helm uninstall "openldap" -n "default"
-    fi
+    uninstallHelmChart openldap default
```

Comment:
FYI, I think this section (or perhaps using `uninstallHelmchart`) resolves https://jts.seagate.com/browse/CORTX-29468, which I submitted earlier this week.

Reply:
I assigned that to myself and will close when this resolves.